pkg/stub/handler.go

```diff
@@ -59,3 +104,20 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 	}
 	return nil
 }
+
+func unknownStorage(typ string) bool {
+	known := []string{
```

Comment:
map/set seems more suitable

Reply:
Map or set? Do you mean, to ensure the list does not contain duplicate entries? As this is a very small static array, I don't see much value in having a map here instead of an array, but perhaps I'm missing something.